Bio/PDB/FragmentMapper.py

```diff
@@ -49,8 +49,7 @@
 
 
 def _read_fragments(size, length, dir="."):
-    """
-    Read a fragment spec file (available from
+    """Read a fragment spec file (available from
```

Comment:
Not a self contained one line summary, how about:

```python
"""Read a fragment spec file (PRIVATE).

Read a fragment spec file available from
http://csb.stanford.edu/rachel/fragments/
and return a list of Fragment objects.
"""
```

Note the URL didn't look to be using RST markup, so fixing that too.